WS2012R2/lisa/setupscripts/TCUtils.ps1

```diff
@@ -1582,23 +1583,59 @@ function GetNumaSupportStatus([string] $kernel)
         GetNumaSupportStatus 2.6.32-696.el6.x86_64
     #>
 
-    if ( $kernel.Contains("i686") -or $kernel.Contains("i386")) {
-        return $false
+    if( $kernel.Contains("i686") `
+        -or $kernel.Contains("i386")){
+            return $false
     }
+    $numaSupport = "2.6.32.504"
+    $kernelSupport = $numaSupport.split(".")
+    $kernelCurrent = $kernel.replace("-",".").split(".")
 
-    if ( $kernel.StartsWith("2.6")) {
```

Comment:
We need to keep this code as we're checking only if we have the 2.6 kernel for NUMA support, otherwise we don't want to process the kernel build